src/components/Personalization/dom-actions/dom/helperForShadow.js

```diff
@@ -0,0 +1,25 @@
+/*
+Copyright 2021 Adobe. All rights reserved.
+This file is licensed to you under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License. You may obtain a copy
+of the License at http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software distributed under
+the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+OF ANY KIND, either express or implied. See the License for the specific language
+governing permissions and limitations under the License.
+*/
+
+const SHADOW_SEPARATOR = ":shadow";
+
+export const isNotShadowSelector = str => str.indexOf(SHADOW_SEPARATOR) === -1;
+
+export const splitWithShadow = selector => {
+  return selector.split(SHADOW_SEPARATOR);
+};
+
+export const cleanupPrefix = selector => {
+  const result = selector.trim();
+  const subselIdx = result.indexOf(">");
+  return subselIdx === 0 ? result.substring(1).trim() : result;
+};
```

Comment:
Sorry, I wasn't aware of the `helperForEq.js` precedent. Moving these particular helpers back to `selectNodesWithShadow` is good though IMO. At this point, I would recommend not exporting or unit testing `splitWithShadow` or `transformPrefix` directly, since those are implementation details that should remain private to `selectNodesWithShadow.js`. Instead, just test the default export of `selectNodesWithShadow.js` in a way that you are sure `splitWithShadow` and `transformPrefix` are correctly performing their job. By not exporting or testing `splitWithShadow` or `transformPrefix` directly, our tests become more resilient to change. In other words, we'll spend less time updating our tests if we need to refactor those functions down the road.

`isShadowSelector` is the oddball here, since that function is reasonably used both internally within `selectNodesWithShadow.js` and in `selectNodes.js`. I didn't realize this when I suggesting moving the functions into `selectNodesWithShadow.js`. It's probably fine where you have it.